docs/_static/js/rst_links_fix.js

```diff
@@ -1,4 +1,5 @@
 $(function() {
     $('a[href^="./"][href*=".rst"]').attr('href', (i, val) => { return val.replace('.rst', '.html'); });  /* Replace '.rst' with '.html' in all internal links like './[Something].rst[#anchor]' */
     $('.wy-nav-content').each(function () { this.style.setProperty('max-width', 'none', 'important'); });
+    $('.wy-menu.wy-menu-vertical > ul:nth-of-type(2)').hide();  /* Fix theme navbar shows hidden toctree */
```

Comment:
what is the second `<ul>` inlcuding? I only find the first `<ul>`with "Contents"

Reply:
`sphinx_rtd_theme` renders a toctree which should be non-visible.
https://github.com/Microsoft/LightGBM/pull/997/files#diff-caf2a6b8f4947d018f68893c695b5202R27

http://www.sphinx-doc.org/en/stable/markup/toctree.html
>This will still notify Sphinx of the document hierarchy, but not insert links into the document at the location of the directive – this makes sense if you intend to insert these links yourself, in a different style, or in the HTML sidebar.

I don't know another way to avoid warnings about documents aren't included into any toctree.

Reply:
From their documentation:
https://github.com/rtfd/sphinx_rtd_theme#how-the-table-of-contents-builds
>Also note that the table of contents is set with includehidden=true. This allows you to set a hidden toc in your index file with the hidden property that will allow you to build a toc without it rendering in your index.